lib/erb_lint/formatters/json_formatter.rb

```diff
@@ -0,0 +1,82 @@
+# frozen_string_literal: true
+
+module ERBLint
+  module Formatters
+    class JSONFormatter
+      RUBOCOP_MESSAGE_PATTERN = /\A(?<linter>\S+): (?<message>.+)\z/.freeze
+
+      def initialize
+        @result = {
+          metadata: metadata_hash,
+          files: [],
+          summary: {offense_count: 0, inspected_file_count: 0}
```

Comment:
It should be interesting to have an offending files counter.

```suggestion
          summary: { offense_count: 0, inspected_file_count: 0, offending_file_count: 0 }
```